### ISSUE TITLE
Allow ignoring the rest of path

### ISSUE THREAD

**Issue by malaire** [elm/url#19]
For my current app I need a parser which only checks that some initial segments of path match the parser, and then ignore the rest of the segments, if any, and regardless of how many there are.

This doesn't seem to be possible currently. Custom parser doesn't help as it would require me to know exactly how many segments I want to ignore, which I don't know.

**Comment by Erudition** [comment]
Yeah that's really custom, sounds like you should just parse it yourself with `String.split "/"`
